.github/workflows/api-check.yml

```diff
@@ -2,6 +2,10 @@ name: API Stability Check
 
 on:
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
This is actually the default when no `types` are specified ([ref](https://docs.github.com/fr/actions/reference/workflows-and-actions/events-that-trigger-workflows#pull_request)).

Any particular reason to explicit that?

```suggestion
```